src/pages/api/sanity/lessons/create.ts

```diff
@@ -75,17 +141,21 @@ const createSanityLessons = async (
     } else {
       let transaction = sanityClient.transaction()
 
-      const {sanityLessons, sanityResources} =
-        await formatSanityMutationForLessons(req.body.lessons)
+      const {course, lessons} = req.body
 
-      sanityResources.forEach((lesson) => {
-        transaction.create(lesson)
+      const {sanityCourse, sanityLessons, sanityResources} =
+        await formatSanityMutationForLessons(course, lessons)
+
+      sanityResources.forEach((resource) => {
+        transaction.create(resource)
       })
 
       sanityLessons.forEach((lesson) => {
         transaction.create(lesson)
       })
 
+      transaction.create(sanityCourse)
```

Comment:
We are now creating the Sanity `course` along with the `lesson` and `videoResource` entities.